core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/ScheduledRepairJobView.java

```diff
@@ -49,48 +54,94 @@ public ScheduledRepairJobView(UUID id, TableReference tableReference, RepairConf
         myCompletionTime = repairStateSnapshot.lastCompletedAt();
     }
 
+    /**
+     * Get id.
+     *
+     * @return UUID
+     */
     public UUID getId()
     {
         return myId;
     }
 
+    /**
+     * Get table reference.
+     *
+     * @return TableReference
+     */
     public TableReference getTableReference()
     {
         return myTableReference;
     }
 
+    /**
+     * Get repair configuration.
+     *
+     * @return RepairConfiguration
+     */
     public RepairConfiguration getRepairConfiguration()
     {
         return myRepairConfiguration;
     }
 
+    /**
+     * Get repair snapshot.
+     *
+     * @return RepairStateSnapshot
+     */
     public RepairStateSnapshot getRepairStateSnapshot()
     {
         return myRepairStateSnapshot;
     }
 
+    /**
+     * Get status.
+     *
+     * @return Status
+     */
     public Status getStatus()
     {
         return myStatus;
     }
 
+    /**
+     * Get progress.
+     *
+     * @return double
+     */
     public double getProgress()
     {
         return myProgress;
     }
 
+    /**
+     * Get next repair.
+     *
+     * @return long
+     */
     public long getNextRepair()
     {
         return myNextRepair;
     }
 
+    /**
+     * Get completion time.
+     *
+     * @return long
+     */
     public long getCompletionTime()
     {
         return myCompletionTime;
     }
 
+    /**
+     * Equality.
```

Comment:
Might be worth to include what variables are considered here, since not all are.